internal/baserpc/interface.go

```diff
@@ -0,0 +1,8 @@
+package baserpc
+
+import "github.com/dwarvesf/icy-backend/internal/model"
+
+type IBaseRpc interface {
```

Comment:
```suggestion
type IBaseRPC interface {
```

either using this or change other variables (see you use ICY, BTC, ...) to this convention